packages/mathjax2-extension/src/index.ts

```diff
@@ -1,42 +0,0 @@
-/* -----------------------------------------------------------------------------
-| Copyright (c) Jupyter Development Team.
-| Distributed under the terms of the Modified BSD License.
-|----------------------------------------------------------------------------*/
-/**
- * @packageDocumentation
- * @module mathjax2-extension
- */
-
-import { JupyterFrontEndPlugin } from '@jupyterlab/application';
-import { PageConfig } from '@jupyterlab/coreutils';
-import { MathJaxTypesetter } from '@jupyterlab/mathjax2';
-import { ILatexTypesetter } from '@jupyterlab/rendermime';
-
-/**
- * The MathJax latexTypesetter plugin.
- */
-const plugin: JupyterFrontEndPlugin<ILatexTypesetter> = {
-  id: '@jupyterlab/mathjax2-extension:plugin',
-  autoStart: true,
-  provides: ILatexTypesetter,
-  activate: () => {
-    const [urlParam, configParam] = ['fullMathjaxUrl', 'mathjaxConfig'];
-    const url = PageConfig.getOption(urlParam);
-    const config = PageConfig.getOption(configParam);
```

Comment:
Some changes must be done on the backend side as we are not using this page config options anymore.

Reply:
Do these page config options live anywhere besides `jupyterlab_server` (JLS)? I'm happy to author the PR to remove these options.

Reply:
Indeed there is the set up in `jupyterlab_server` at https://github.com/jupyterlab/jupyterlab_server/blob/fb77c74f022d439249f4529687db1582bfe282b3/jupyterlab_server/handlers.py#L108

Could you also open an issue on [jupyter_server_mathjax](https://github.com/jupyter-server/jupyter_server_mathjax) repository as it won't work with JupyterLab 4?

Reply:
Can these be done after this PR is merged? Or do the page config options need to be removed in JLS before upgrading to MathJax 3 in JL?

Reply:
@fcollonval

Reply:
Yes there is no problem doing that in follow-up.